WcaOnRails/app/views/competitions/_nav.html.erb

```diff
@@ -275,6 +275,16 @@
         <%= alert :info, message, note: true %>
       <% end %>
 
+      <%= alert :info, note: true do %>
```

Comment:
Why are you creating an extra div here, instead of just adding the raw `competitions.messages.in_progress_at_wca_live_html` into the `info` array above?

See also my comment in `competition.rb`